src/_includes/main-intro/style.css

```
@@ -1,5 +1,181 @@
-.title {
-  font-size: 50px;
+.root {
+  overflow: hidden;
+  position: relative;
+  line-height: normal;
+  background: var(--offwhite);
+  padding: 4.4rem 0;
+
+  @media (min-width: 720px) {
+    height: 90vh;
+    display: flex;
+    align-items: center;
+    background: linear-gradient(to bottom, #fff, var(--offwhite));
+    padding: 0;
+  }
+}
+
+.conf-detail {
+  text-align: center;
+  margin: 0 var(--side-margin);
+  color: var(--darkgrey);
+  position: relative;
+
+  @media (min-width: 720px) {
+    text-align: left;
+    font-size: 1.25rem;
+  }
+}
+
+.month {
+  text-transform: lowercase;
+}
+
+.date {
   font-family: 'Google Sans', roboto, sans-serif;
   font-weight: bold;
+  font-size: 2.14rem;
+  display: block;
+  margin: 0.25em 0;
+  line-height: 1;
+
+  @media (min-width: 720px) {
+    font-size: 3.24rem;
+  }
+}
+
+.location-main {
+  display: block;
+}
+
+.location-sub {
+  display: block;
+  font-weight: 300;
+  margin: 0.35em 0;
+
+  @media (min-width: 720px) {
+    margin: 0.2em 0;
+  }
+}
+
+.date-link {
+  white-space: nowrap;
+  text-decoration: none;
+}
+
+.actions {
+  @media (min-width: 720px) {
+    margin: 5.9rem 0 4rem;
+    display: grid;
+    grid-auto-columns: min-content;
+    font-size: 0.75rem;
+    grid-auto-flow: column;
+    align-items: start;
+    grid-gap: 1.6em;
+  }
+}
+
+.register {
+  text-decoration: none;
+  color: #fff;
+  background: var(--link-color);
+  border-radius: 1000px;
+  padding: 0.9em 3em;
+  display: block;
+  box-shadow: 0 3px 16px var(--blue-a25);
+  display: none;
+
+  @media (min-width: 720px) {
+    display: block;
+  }
+}
+
+.cal {
+  padding: 0.9em 0;
+  display: block;
+  position: relative;
+  white-space: nowrap;
+
+  @media (min-width: 720px) {
+    --icon-size: 30px;
```

What about this instead?

```suggestion
    --icon-size: 1.5rem;
```